docs/getting-started.md

```diff
@@ -114,6 +114,16 @@ Console.ReadLine();
 host.Stop();
 ```
 
+## Registry
+
+The workflow host keeps workflow definitions in the registry when they are registered.  When starting a workflow, the workflow host creates a new instance for the desired workflow definition.
+
+
+## Persistence
+
+The persistence provider persists the state of a workflow instance with execution pointers.  They are created when visiting a step while executing a workflow and hold information about their outcome.
```

Comment:
I think execution pointers are more of an internal concept, not really specific to persistence?
Either way, I'm not sure this paragraph gives a description of what they are.